backend/prometheus_app/dockerfile

```diff
@@ -0,0 +1,24 @@
+# Image Python officielle
+FROM python:3.11-slim
+
+# Définir le dossier de travail
+WORKDIR /app
+
+# Copier le fichier requirements.txt depuis la racine du projet
+COPY requirements.txt .
+
+# Installer les dépendances Python
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copier le contenu du dossier prometheus_app dans /app
+COPY ./backend/prometheus_app /app
+
+# Exposer le port FastAPI
+EXPOSE 8000
+
+# Lancer le serveur FastAPI
+CMD ["uvicorn", "core_services_monitoring:app", "--host", "0.0.0.0", "--port", "8000"]
+
+
+#docker build -t oai-backend -f dockerfile .
```

Comment:
remove those command. Put in documentation if you want to explain how to built it.